experiments/simbricks/utils/graphlib.py

```diff
@@ -1,14 +1,14 @@
 # From cpython repo (introduced upstream in python 3.9)
```

Comment:
Graphlib sollten wir vielleicht vom ganzen Linting ausschliessen. Das ist ja direkt aus dem python repo kopiert da das zumindest eine Zeit lang noch nicht in unserer verwendeten python version drin war. Da würde ich gerne die originale Formatierung beibehalten.